bin/propolis-standalone/README.md

```diff
@@ -23,6 +23,9 @@ memory = 1024
 # Exit propolis-standalone process with <code> if instance reboots (default: unset)
 # exit_on_reboot = <code>
 
+# Override boot order (via communication to OVMF bootrom)
```

Comment:
nitpicky, not a big deal: maybe worth explicitly noting here that this is an ordered list of device names (and not `block_dev` names)?

Reply:
I thought that this was clear contextually, both with the inclusion of `net0`, and that `block0`'s backend is not named the same.

Reply:
Yeah, that's fair!